Alternate-contacts: Bug in latest jsonschema package

Hello,

While deploying the **aft-alternate-contacts** solution I ran into an issue in the **validate-alternate-contacts** Lambda function:

`
[ERROR] Runtime.ImportModuleError: Unable to import module 'validate-alternate-contacts': No module named 'rpds.rpds' Traceback (most recent call last):
`

This is a known problem in the **jsconschema** package. installed version was 4.19.X. This is a known issue by the provider: https://github.com/python-jsonschema/jsonschema/issues/1117

Setting the version to **4.17.3** in the **requirements.txt** and redeploying solved the issue.

`jsonschema==4.17.3`

Can you set the version constraint in the module until the issue is fixed by the provider?

--- a/lambda/aft_alternate_contacts_validate/requirements.txt
+++ b/lambda/aft_alternate_contacts_validate/requirements.txt
@@ -1,1 +1,1 @@
-jsonschema
+jsonschema==4.17.3

